src/README.md

```diff
@@ -0,0 +1,53 @@
+![Storyblok ImagoType](https://raw.githubusercontent.com/storyblok/.github/refs/heads/main/profile/public/github-banner.png)
+
+# Storyblok CLI Package Reference
+
+## Installation
+
+For the latest beta version, install the package using the following command:
+
+```bash
+npm install storyblok@beta
+```
+
+Or for an specific beta version:
+
+```bash
+npm install storyblok@4.0.0-beta.<version>
+```
+
+## Command Reference
+
+| Command | Status | Notes |
+|---------|--------|-------|
+| [`login`](./commands/login/README.md) | ✅ Ready | Improved DX and credentials storage in ~/.storyblok/credentials.json |
+| [`logout`](./commands/logout/README.md) | ✅ Ready | |
+| [`user`](./commands/user/README.md) | ✅ Ready | |
+| [`languages pull`](./commands/languages/README.md) | ✅ Ready | Replaces previous pull-languages |
+| [`components pull`](./commands/components/pull/README.md) | ✅ Ready | Replaces previous pull-components |
+| [`components push`](./commands/components/push/README.md) | ✅ Ready | Replaces previous push-components. Also handles dependencies such as groups, tags, presets and whitelists. (Datasources is pending) |
+| `components delete` | 📝 Planned | Will replace delete-component and delete-components |
+| [`migrations generate`](./commands/migrations/generate/README.md) | ✅ Ready | Replaces previous generate-migrations |
+| [`migrations run`](./commands/migrations/run/README.md) | ✅ Ready | Replaces previous run-migrations |
+| [`migrations rollback`](./commands/migrations/rollback/README.md) | ✅ Ready | Replaces previous rollback-migrations |
+| [`types generate`](./commands/types/generate/README.md) | ✅ Ready | Replaces previous generate-typescript-typedefs |
+| `sync` | ⚠️ v3 | |
```

Comment:
I suggest leaving only the emojis, and adding a legend in the end or before the table.

Reply:
@jeremenichelli I'm also not a fan of emojis, but I think they serve a purpose here.